Description: force build of qtsensors and qtserialport
Author: Dmitry Shachnev <mitya57@ubuntu.com>
Forwarded: not-needed
Last-Update: 2013-01-10

--- a/configure.py
+++ b/configure.py
@@ -1003,8 +1003,7 @@
             'QDBusConnection::systemBus()')
     check_dbus(target_config, verbose)
 
-    if target_config.qt_version >= 0x050100:
-        check_5_1_modules(target_config, verbose)
+    check_5_1_modules(target_config, verbose)
 
     if target_config.qt_version >= 0x050200:
         check_5_2_modules(target_config, verbose)
