Description: add support for Qt 5.9
Origin: upstream, from 5.8 and 5.9 releases
Last-Update: 2017-07-08

--- a/sip/QtCore/QtCoremod.sip
+++ b/sip/QtCore/QtCoremod.sip
@@ -22,7 +22,7 @@
 
 %Module(name=PyQt5.QtCore, call_super_init=True, default_VirtualErrorHandler=PyQt5, keyword_arguments="Optional", version=1)
 
-%Timeline {Qt_5_0_0 Qt_5_0_1 Qt_5_0_2 Qt_5_1_0 Qt_5_1_1 Qt_5_2_0 Qt_5_2_1 Qt_5_3_0 Qt_5_3_1 Qt_5_3_2 Qt_5_4_0 Qt_5_4_1 Qt_5_4_2 Qt_5_5_0 Qt_5_5_1 Qt_5_6_0 Qt_5_6_1 Qt_5_7_0}
+%Timeline {Qt_5_0_0 Qt_5_0_1 Qt_5_0_2 Qt_5_1_0 Qt_5_1_1 Qt_5_2_0 Qt_5_2_1 Qt_5_3_0 Qt_5_3_1 Qt_5_3_2 Qt_5_4_0 Qt_5_4_1 Qt_5_4_2 Qt_5_5_0 Qt_5_5_1 Qt_5_6_0 Qt_5_6_1 Qt_5_7_0 Qt_5_7_1 Qt_5_8_0 Qt_5_8_1 Qt_5_9_0 Qt_5_9_1}
 
 %Platforms {WS_X11 WS_WIN WS_MACX}
 
--- a/sip/QtCore/qnamespace.sip
+++ b/sip/QtCore/qnamespace.sip
@@ -209,8 +209,6 @@
         WindowContextHelpButtonHint,
         WindowShadeButtonHint,
         WindowStaysOnTopHint,
-        WindowOkButtonHint,
-        WindowCancelButtonHint,
         WindowStaysOnBottomHint,
         WindowCloseButtonHint,
         MacWindowToolBarButtonHint,
--- a/sip/QtCore/qobject.sip
+++ b/sip/QtCore/qobject.sip
@@ -527,8 +527,8 @@
     void setParent(QObject * /TransferThis/);
     void installEventFilter(QObject *);
     void removeEventFilter(QObject *);
-    void dumpObjectInfo();
-    void dumpObjectTree();
+    void dumpObjectInfo() const;
+    void dumpObjectTree() const;
     QList<QByteArray> dynamicPropertyNames() const;
     bool setProperty(const char *name, const QVariant &value);
     QVariant property(const char *name) const;
--- a/sip/QtQml/qqmlapplicationengine.sip
+++ b/sip/QtQml/qqmlapplicationengine.sip
@@ -33,7 +33,7 @@
     QQmlApplicationEngine(const QUrl &url, QObject *parent /TransferThis/ = 0);
     QQmlApplicationEngine(const QString &filePath, QObject *parent /TransferThis/ = 0);
     virtual ~QQmlApplicationEngine();
-    QList<QObject *> rootObjects();
+    QList<QObject *> rootObjects() const;
 
 public slots:
     void load(const QUrl &url) /ReleaseGIL/;
--- a/sip/QtQuick/qquickitemgrabresult.sip
+++ b/sip/QtQuick/qquickitemgrabresult.sip
@@ -31,7 +31,7 @@
 public:
     QImage image() const;
     QUrl url() const;
-    bool saveToFile(const QString &fileName);
+    bool saveToFile(const QString &fileName) const;
 
 protected:
     virtual bool event(QEvent *);
--- a/sip/QtSql/qsql.sip
+++ b/sip/QtSql/qsql.sip
@@ -23,7 +23,7 @@
 namespace QSql
 {
 %TypeHeaderCode
-#include <qsql.h>
+#include <qtsqlglobal.h>
 %End
 
     enum Location
